config/settings/monitor.q

```diff
@@ -1,5 +1,15 @@
 // Default configuration for the monitor process
 
+\d .monitor 
+configcsv: first .proc.getconfigfile["monitorconfig.csv"];
```

Comment:
Can you add a comment in here for what each of these variables are i.e. what they control